README.md

```diff
@@ -1,15 +1,20 @@
-[ci-img]: https://img.shields.io/travis/ciena-blueplanet/ember-test-utils.svg "Travis CI Build Status"
-[ci-url]: https://travis-ci.org/ciena-blueplanet/ember-test-utils
+# ember-test-utils
 
-[cov-img]: https://img.shields.io/coveralls/ciena-blueplanet/ember-test-utils.svg "Coveralls Code Coverage"
-[cov-url]: https://coveralls.io/github/ciena-blueplanet/ember-test-utils
+Ember testing utilities.
 
-[npm-img]: https://img.shields.io/npm/v/ember-test-utils.svg "NPM Version"
-[npm-url]: https://www.npmjs.com/package/ember-test-utils
+###### Dependencies
```

Comment:
I never really liked these sections, they take up so much vertical space. Not sure of a better idea though, except maybe cutting out some of the badges...